cocos/2d/CCFontAtlas.cpp

```diff
@@ -153,6 +146,16 @@ FontAtlas::~FontAtlas()
 #endif
 }
 
+void FontAtlas::initTextureWithZeros(Texture2D *texture)
+{
+    char *zeros = nullptr;
+    size_t cnt = _currentPixelFormat == Texture2D::PixelFormat::RGBA8888 ? CacheTextureWidth * CacheTextureWidth * 4 : CacheTextureWidth * CacheTextureWidth;
+    zeros = new char[cnt];
+    std::fill(zeros, zeros + cnt, 0);
+    texture->initWithData(zeros, cnt, _currentPixelFormat, CacheTextureWidth, CacheTextureHeight, Size(CacheTextureWidth, CacheTextureHeight));
+    delete[] zeros;
+}
```

Comment:
I think this function make codes not clear as before.

Reply:
This procedure is also invoked in `prepareLetterDefinitions()`.
Is it a naming issue?